mindsdb_native/libs/backends/lightwood.py

```diff
@@ -171,7 +171,7 @@ def train(self):
         if self.transaction.lmd['use_gpu'] is not None:
             lightwood.config.config.CONFIG.USE_CUDA = self.transaction.lmd['use_gpu']
 
-        if self.transaction.lmd['model_order_by'] is not None and len(self.transaction.lmd['model_order_by']) > 0:
+        if self.transaction.lmd['tss']['order_by'] is not None and len(self.transaction.lmd['tss']['order_by']) > 0:
```

Comment:
2. so this check for None is redundant]
(see previous comment)

Reply:
This check should be changed to `self.transaction.lmd['tss']['is_timeseries']`